source/common/quic/quic_filter_manager_connection_impl.h

```diff
@@ -77,7 +77,7 @@ class QuicFilterManagerConnectionImpl : public Network::ConnectionImplBase,
   absl::optional<Network::Connection::UnixDomainSocketPeerCredentials>
   unixSocketPeerCredentials() const override {
     // Unix domain socket is not supported.
-    NOT_REACHED_GCOVR_EXCL_LINE;
+    return absl::nullopt;
```

Comment:
I'm having trouble finding non-test uses of unixSocketPeerCredentials().

What am I missing?  Who depends on this API?

Reply:
Not sure. I'd be happy to remove as a follow-up if we think it's doable.
I wonder if it's used internally by whoever @snowp worked for at the time?